targets/mbedos5/README.md

```diff
@@ -50,6 +50,15 @@ jshint is used to statically check your JavaScript code, as part of the build pr
 This ensures that pins you are using in your code are available on your chosen target
 platform.
 
+### Python modules
+
+There are Python module dependencies which can be installed with the following command: 
+	`pip install -r requirements.txt`
+Use this command in your .../jerryscript/targets/mbedos5/tools directory. 
```

Comment:
If you want this command to stand out on its own then this is not the proper markdown syntax. Right now, the pip command is inlined. Either add extra empty lines before and after the command (and remove the backticks from around the command), or add triple-backticks around the command (but don't indent it then) (as seen in the Quick Start section).

Moreover, I think you have used TAB for indenting. Please, don't do that. TABs are evil. Use spaces (if needed).